app.js

```diff
@@ -1,98 +1,68 @@
 // --------------------------------- 80chars ---------------------------------->
 
-import express from 'express';
```

Comment:
looking at javascript without semicolons freaks me out haha, but i will adjust

Reply:
Ha, I'm willing to lose that fight. :) Whoever's writing the code gets to decide the conventions. (So I shall stop removing semicolons willy-nilly and only do so when I'm actually contributing things to the code!)

Reply:
i'd rather see it all consistent than have my way, so let's just go _commando_ heh